### ISSUE TITLE
Help 

### ISSUE THREAD

**Issue by jcolbert89** [Protocentral/AFE4490_Oximeter#5]
Hi guys, I'm having trouble getting this to work. I followed the instructions as best as I could. Unfortunately I'm not getting any readings. When I click the run button, I'm able to select the port and press start but I do not get any readings. The RED PPG is a flat green light while the IR doesn't register. No SpO2 value is displayed.

I'd be grateful if you could help me as I am inexperienced with such projects.

**Comment by samuelxu91** [comment]
I downloaded AFE4490_Oximeter/gui/openview_pulseox.zip on Apr 14, and I also couldn't see any readings after I select the port and press start. However, it worked when I change
_if( CES_Pkt_Len  == 9)_
into
_if( CES_Pkt_Len  == 10)_
in openview_pulseox.pde file
The line is in the switch-case block of _case CESState_PktLen_Found_. It should be somewhere between line 270 and 290.

I looked at the directory AFE4490_Oximeter/gui/openview_pulseox but I didn't see such error. Maybe the developers have fixed the problem in the code in that directory, but the zip file is not up to date.

**Comment by adelalfusail** [comment]
did you fix this problem or not

**Comment by samuelxu91** [comment]
No, I didn't. I am new to GitHub. Am I able to modify the files written by other people?

**Comment by adelalfusail** [comment]
Your mean problem that you did not update your files. protocentral is updated openview_pulseox.pde file
and change it to if( CES_Pkt_Len == 10)
